src/backend/controllers/userController.js

```diff
@@ -1,24 +1,20 @@
 // @flow strict
 
 // Required imports
+import type { $Request, $Response } from 'express';
+import nodemailer from 'nodemailer';
+import crypto from 'crypto';
+
 const bcrypt = require('bcrypt');
 const jwt = require('jsonwebtoken');
 const { validationResult } = require('express-validator');
-const crypto = require('crypto');
-const nodemailer = require('nodemailer');
-import type {
-	$Request,
-	$Response,
-	NextFunction,
-	Middleware,
-} from 'express';
+// Throws an error if this isn't here because of async functions
+const regeneratorRuntime = require("regenerator-runtime");
```

Comment:
is this used?

Reply:
Yeah I haven't really figured out why only this controller requires it while the others don't.

Reply:
hmm. Odd. Well if you never use the variable, you can change the line to just:
`require("regenerator-runtime");`